### DISCUSSION TITLE
many security onion so services show starting state

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#7693] [Unsupported Versions]
![5566best](https://user-images.githubusercontent.com/41321407/161363324-b26e8d18-3d4b-42ed-9959-c2436bdf098e.PNG)

when I reboot my computer, many security onion services show STARTING and I can not open the security onion console interface.

**Comment by dougburks** [comment]
As mentioned at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720, please provide more information so that we can help you:
>What version of Security Onion are you using?
Did you install from our Security Onion ISO image or did you perform a network installation?
If network installation, did you install on CentOS 7 or Ubuntu?
Do you get any failures when you run sudo salt-call state.highstate?
What are the specs of the machine?

**Reply by D4thToMS** [reply]
I have the same issue On CentOS7  Security Onion 2.3.110, installed from ISO a year ago then updated ever since.
I ran sudo salt-call state.highstate:
`[ERROR   ] 'pkg.unhold' failed: Cannot proceed, python2-dnf-plugin-versionlock is not installed` I installed it using dnf, then rebooted and nothing changed, still only a few services up:
![Screenshot 2022-04-04 at 13 58 40](https://user-images.githubusercontent.com/78807384/161539137-aaf4aca6-78a0-4ae1-8da7-e5b307f8a857.png)

**Reply by D4thToMS** [reply]
Machine is Dell Optiplex 7010 (16GB RAM i5-3470 256GB SSD+2TB Sata)
`$ sudo salt-call state.highstate` looks nice too:

`local:
----------
          ID: minion-state-apply-test
    Function: file.touch
        Name: /opt/so/log/salt/state-apply-test
      Result: True
     Comment: Updated times on file /opt/so/log/salt/state-apply-test
     Started: 12:09:24.151864
    Duration: 10.415 ms
     Changes:
              ----------
              touched:
                  /opt/so/log/salt/state-apply-test
----------
          ID: crond_service
    Function: service.running
        Name: crond
      Result: True
     Comment: The service crond is already running
     Started: 12:09:24.179214
    Duration: 1544.886 ms
     Changes:
----------
          ID: CentOS-Base.repo
    Function: file.absent
        Name: /etc/yum.repos.d/CentOS-Base.repo
      Result: True
     Comment: File /etc/yum.repos.d/CentOS-Base.repo is not present
     Started: 12:09:25.724633
    Duration: 2.475 ms
     Changes:
----------
          ID: CentOS-CR.repo
    Function: file.absent
        Name: /etc/yum.repos.d/CentOS-CR.repo
      Result: True
     Comment: File /etc/yum.repos.d/CentOS-CR.repo is not present
     Started: 12:09:25.727383
    Duration: 1.969 ms
     Changes:
----------
          ID: CentOS-Debuginfo.repo
    Function: file.absent
        Name: /etc/yum.repos.d/CentOS-Debuginfo.repo
      Result: True
     Comment: File /etc/yum.repos.d/CentOS-Debuginfo.repo is not present
     Started: 12:09:25.729595
    Duration: 2.022 ms
     Changes:
----------
          ID: CentOS-fasttrack.repo
    Function: file.absent
        Name: /etc/yum.repos.d/CentOS-fasttrack.repo
      Result: True
     Comment: File /etc/yum.repos.d/CentOS-fasttrack.repo is not present
     Started: 12:09:25.731861
    Duration: 1.999 ms
     Changes:
----------
          ID: CentOS-Media.repo
    Function: file.absent
        Name: /etc/yum.repos.d/CentOS-Media.repo
      Result: True
     Comment: File /etc/yum.repos.d/CentOS-Media.repo is not present
     Started: 12:09:25.734113
    Duration: 1.936 ms
     Changes:
----------
          ID: CentOS-Sources.repo
    Function: file.absent
        Name: /etc/yum.repos.d/CentOS-Sources.repo
      Result: True
     Comment: File /etc/yum.repos.d/CentOS-Sources.repo is not present
     Started: 12:09:25.736290
    Duration: 1.853 ms
     Changes:
----------
          ID: CentOS-Vault.repo
    Function: file.absent
        Name: /etc/yum.repos.d/CentOS-Vault.repo
      Result: True
     Comment: File /etc/yum.repos.d/CentOS-Vault.repo is not present
     Started: 12:09:25.738398
    Duration: 1.98 ms
     Changes:
----------
          ID: CentOS-x86_64-kernel.repo
    Function: file.absent
        Name: /etc/yum.repos.d/CentOS-x86_64-kernel.repo
      Result: True
     Comment: File /etc/yum.repos.d/CentOS-x86_64-kernel.repo is not present
     Started: 12:09:25.740612
    Duration: 1.852 ms
     Changes:
----------
          ID: docker-ce.repo
    Function: file.absent
        Name: /etc/yum.repos.d/docker-ce.repo
      Result: True
     Comment: File /etc/yum.repos.d/docker-ce.repo is not present
     Started: 12:09:25.742690
    Duration: 1.788 ms
     Changes:
----------
          ID: epel.repo
    Function: file.absent
        Name: /etc/yum.repos.d/epel.repo
      Result: True
     Comment: File /etc/yum.repos.d/epel.repo is not present
     Started: 12:09:25.744698
    Duration: 1.748 ms
     Changes:
----------
          ID: epel-testing.repo
    Function: file.absent
        Name: /etc/yum.repos.d/epel-testing.repo
      Result: True
     Comment: File /etc/yum.repos.d/epel-testing.repo is not present
     Started: 12:09:25.746667
    Duration: 1.756 ms
     Changes:
----------
          ID: saltstack.repo
    Function: file.absent
        Name: /etc/yum.repos.d/saltstack.repo
      Result: True
     Comment: File /etc/yum.repos.d/saltstack.repo is not present
     Started: 12:09:25.748641
    Duration: 1.767 ms
     Changes:
----------
          ID: salt-latest.repo
    Function: file.absent
        Name: /etc/yum.repos.d/salt-latest.repo
      Result: True
     Comment: File /etc/yum.repos.d/salt-latest.repo is not present
     Started: 12:09:25.750628
    Duration: 1.76 ms
     Changes:
----------
          ID: wazuh.repo
    Function: file.absent
        Name: /etc/yum.repos.d/wazuh.repo
      Result: True
     Comment: File /etc/yum.repos.d/wazuh.repo is not present
     Started: 12:09:25.752610
    Duration: 1.753 ms
     Changes:
----------
          ID: repair_yumdb
    Function: cmd.run
        Name: mv -f /var/lib/rpm/__db* /tmp && yum clean all
      Result: True
     Comment: onlyif condition is false
     Started: 12:09:25.754587
    Duration: 1110.819 ms
     Changes:
----------
          ID: crsynckeys
    Function: file.recurse
        Name: /etc/pki/rpm_gpg
      Result: True
     Comment: The directory /etc/pki/rpm_gpg is in the correct state
     Started: 12:09:26.866124
    Duration: 191.976 ms
     Changes:
----------
          ID: remove_securityonionrepocache
    Function: file.absent
        Name: /etc/yum.repos.d/securityonioncache.repo
      Result: True
     Comment: File /etc/yum.repos.d/securityonioncache.repo is not present
     Started: 12:09:27.058382
    Duration: 1.994 ms
     Changes:
----------
          ID: crsecurityonionrepo
    Function: file.managed
        Name: /etc/yum.repos.d/securityonion.repo
      Result: True
     Comment: File /etc/yum.repos.d/securityonion.repo is in the correct state
     Started: 12:09:27.060608
    Duration: 12.897 ms
     Changes:
----------
          ID: yumconf
    Function: file.managed
        Name: /etc/yum.conf
      Result: True
     Comment: File /etc/yum.conf is in the correct state
     Started: 12:09:27.073737
    Duration: 24.221 ms
     Changes:
----------
          ID: cleanairgap
    Function: file.absent
        Name: /etc/yum.repos.d/airgap_repo.repo
      Result: True
     Comment: File /etc/yum.repos.d/airgap_repo.repo is not present
     Started: 12:09:27.098195
    Duration: 1.598 ms
     Changes:
----------
          ID: cleanyum
    Function: cmd.run
        Name: yum clean metadata
      Result: True
     Comment: State was not run because none of the onchanges reqs changed
     Started: 12:09:27.105941
    Duration: 0.004 ms
     Changes:
----------
          ID: salt_bootstrap
    Function: file.managed
        Name: /usr/sbin/bootstrap-salt.sh
      Result: True
     Comment: File /usr/sbin/bootstrap-salt.sh is in the correct state
     Started: 12:09:27.106050
    Duration: 19.569 ms
     Changes:
----------
          ID: remove_salt-2019-2-5.repo
    Function: file.absent
        Name: /etc/yum.repos.d/salt-2019-2-5.repo
      Result: True
     Comment: File /etc/yum.repos.d/salt-2019-2-5.repo is not present
     Started: 12:09:27.125833
    Duration: 1.601 ms
     Changes:
----------
          ID: patch_package
    Function: pkg.installed
        Name: patch
      Result: True
     Comment: All specified packages are already installed
     Started: 12:09:27.155791
    Duration: 36.402 ms
     Changes:
----------
          ID: systemd_reload
    Function: module.run
      Result: True
     Comment: service.systemctl_reload: True
     Started: 12:09:27.192374
    Duration: 90.547 ms
     Changes:
              ----------
              service.systemctl_reload:
                  True
----------
          ID: unhold_salt_packages
    Function: module.run
      Result: True
     Comment: pkg.unhold: Success
     Started: 12:09:27.283772
    Duration: 792.541 ms
     Changes:
              ----------
              pkg.unhold:
                  ----------
                  salt:
                      ----------
                      changes:
                          ----------
                      comment:
                          Package salt is not being held.
                      name:
                          salt
                      result:
                          True
                  salt-master:
                      ----------
                      changes:
                          ----------
                      comment:
                          Package salt-master is not being held.
                      name:
                          salt-master
                      result:
                          True
                  salt-minion:
                      ----------
                      changes:
                          ----------
                      comment:
                          Package salt-minion is not being held.
                      name:
                          salt-minion
                      result:
                          True
----------
          ID: install_salt_minion
    Function: cmd.run
        Name: exec 0>&- # close stdin
exec 1>&- # close stdout
exec 2>&- # close stderr
nohup /bin/sh -c 'yum clean all ; /usr/sbin/bootstrap-salt.sh -s 120 -r -F -x python3 stable 3004' &

      Result: True
     Comment: Command "exec 0>&- # close stdin
              exec 1>&- # close stdout
              exec 2>&- # close stderr
              nohup /bin/sh -c 'yum clean all ; /usr/sbin/bootstrap-salt.sh -s 120 -r -F -x python3 stable 3004' &
              " run
     Started: 12:09:28.076824
    Duration: 12.431 ms
     Changes:
              ----------
              pid:
                  3446
              retcode:
                  0
              stderr:
              stdout:
----------
          ID: mine_functions
    Function: file.managed
        Name: /etc/salt/minion.d/mine_functions.conf
      Result: True
     Comment: File /etc/salt/minion.d/mine_functions.conf is in the correct state
     Started: 12:09:28.089748
    Duration: 21.26 ms
     Changes:
----------
          ID: salt_minion_service
    Function: service.running
        Name: salt-minion
      Result: True
     Comment: onlyif condition is false
     Started: 12:09:28.111268
    Duration: 28.678 ms
     Changes:
----------
          ID: patch_pkg
    Function: pkg.installed
        Name: patch
      Result: True
     Comment: All specified packages are already installed
     Started: 12:09:28.140374
    Duration: 39.531 ms
     Changes:

Summary for local
-------------
Succeeded: 32 (changed=4)
Failed:     0
-------------
Total states run:     32
Total run time:    3.968 s`

**Reply by dougburks** [reply]
@D4thToMS From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
